feature-utils/poly-analysis/utils/status.js

```diff
@@ -0,0 +1,23 @@
+export function createSuccessStatus() {
+    return {
+        name: "SUCCESS",
+        isSuccess: true,
+    };
+}
+
+export function createWarningStatus(message) {
+    return {
+        name: "WARNING",
+        message,
+        isWarning: true,
```

Comment:
Also a bit redundant. Essentially, we have status with or without message; I am not totally sure about the `error` attribute below, but I guess it can also be applied to warnings. Besides, we're calling three different functions for different statuses, with not a lot of business logic (other than extracting the name out of the warning). Can we maybe unify it in a single createStatus, or a Status class?

Reply:
yes indeed, a lot of redundancies here, made a separate ticket to refactor this module :)